base-image/execute-tests.sh

```diff
@@ -139,18 +155,40 @@ chmod +x run.sh
 popd
 echo "--------End test--------"
 
+POST_RUN_FILE="${repo_name}"/load-tests/"$scenario_name"/scripts/post_run.sh
+if test -f "$POST_RUN_FILE"; then
+  executeScript "post_run.sh"
+else
+  echo "--------Processing Results--------"
+  pushd "${repo_name}"/load-tests/"$scenario_name"/results/
+  echo "--------Splitting Results--------"
+  jtl-splitter.sh -- -f original.jtl -t 120 -u SECONDS -s
+  ls -ltr
+  echo "--------Splitting Completed--------"
+
+  echo "--------Generating CSV--------"
+  JMeterPluginsCMD.sh --generate-csv summary.csv --input-jtl original-measurement.jtl --plugin-type AggregateReport
```

Comment:
shall we change summary.csv to new_summary.csv as the original summary is there?